File: 프로그래머스/0/120837. 개미 군단/개미 군단.java
class Solution {
    public int solution(int hp) {
        int ant1 = hp / 5;
        int ant2 = (hp % 5) / 3;
        int ant3 = (hp % 5) % 3;
        
        int answer = ant1 + ant2 + ant3;
        return answer;
    }
}

File: 프로그래머스/0/120837. 개미 군단/개미 군단.go
func solution(hp int) int {
    var ant1 int = hp / 5;
    var ant2 int = (hp % 5) / 3;
    var ant3 int = (hp % 5) % 3;
        
    var answer int = ant1 + ant2 + ant3;
    return answer;
}